### PR TITLE
doc: remove expired domain from README

### PR DESCRIPTION
**Pull request by andrewthetechie**
This domain expired and is now being used to host an ai spam blog

### CODE REVIEW

**Comment by andrewthetechie** [comment]
@Kludex thanks for taking ownership of this repo. Figured I'd do a quick fix on the readme to not link to the ai spam site

**Comment by Kludex** [comment]
@andrewthetechie Can you replace it by https://mangum.fastapiexpert.com/?

**Comment by andrewthetechie** [comment]
@Kludex done

File: README.md [hunk 1]
```diff
@@ -10,8 +10,7 @@
 
 Mangum is an adapter for running [ASGI](https://asgi.readthedocs.io/en/latest/) applications in AWS Lambda to handle Function URL, API Gateway, ALB, and Lambda@Edge events.
 
-***Documentation***: https://mangum.io/
-
+***Documentation***: https://mangum.fastapiexpert.com/
```
Comment (Kludex) [inline README.md:13]:
```suggestion
***Documentation***: https://mangum.fastapiexpert.com/

```

**Comment by Kludex** [comment]
Thanks. 👍

**Pull request merged**